packages/core/src/telemetry/anonymous-meta.ts

```diff
@@ -0,0 +1,44 @@
+import os from "os";
+
+type AnonymousMeta = {
+  systemPlatform: NodeJS.Platform;
+  systemRelease: string;
+  systemArchitecture: string;
+  cpuCount: number;
+  cpuModel: string | null;
+  cpuSpeed: number | null;
+  memoryInMb: number;
+  // TODO: detect docker
+  // isDocker: boolean;
+  // isNowDev: boolean;
+  // isWsl: boolean;
+  // isCI: boolean;
+  // ciName: string | null;
+  // nextVersion: string;
+};
+
+let traits: AnonymousMeta | undefined;
+
+export function getAnonymousMeta(): AnonymousMeta {
+  if (traits) {
+    return traits;
+  }
+
+  const cpus = os.cpus() || [];
```

Comment:
Probably doesn't matter here, but interesting to see that this is a very slow synchronous function. https://twitter.com/jarredsumner/status/1683710506924933120?s=20